README.md

```diff
@@ -84,6 +84,7 @@ This file contains a number of front-end interview questions that can be used wh
 * What are the advantages/disadvantages of using CSS preprocessors?
   * Describe what you like and dislike about the CSS preprocessors you have used.
 * How would you implement a web design comp that uses non-standard fonts?
+* Explain the advantages and disadvantages of using icon fonts vs SVG.
```

Comment:
What about:
> Explain why you might us SVG instead of icon fonts or vice-versa?
It's a bit less of a leading question and doesn't require someone to think about the "disadvantages" so much as the "advantages" of one thing over another.